modules/page/html/render/public.js

```diff
@@ -267,11 +269,28 @@ exports.create = function (api) {
 
       var shownWhen = computed([loading, contact.isNotFollowingAnybody],
            (isLoading, isNotFollowingAnybody) => !isLoading && isNotFollowingAnybody
-        )
+      )
 
       return api.feed.html.followWarning(shownWhen, explanation)
     }
 
+    function noFollowersWarning () {
+      var explanation = i18n(
```

Comment:
I hope at some point we can start transitioning to `const` over `var`. Though not necessarily in this change.